core/src/commonMain/kotlin/ua/syt0r/kanji/core/PathExtensions.kt

```diff
@@ -45,6 +46,29 @@ fun Path.approximateEvenly(pointsCount: Int): ApproximatedPath {
     )
 }
 
+fun Path.approximateEquidistant(distance: Float): ApproximatedPath {
+    val pathMeasure = PathMeasure()
+    pathMeasure.setPath(this, false)
+
+    val pathLength = pathMeasure.length
+
+    // less than "distance" pixels at the end are discarded due to rounding
+    val nIntervals = max(1, (pathLength / distance).toInt())
+
+    // divide path into nIntervals segments, store all starting points of all segments and
+    // the end point of the last segment.
+    val points = (0 until  nIntervals+1).map {
+        val fraction = it.toFloat() / nIntervals.toFloat() * pathLength
```

Comment:
Made min number of intervals 1 since there is division by 0 herem does it make sence?

Reply:
Good catch! Didn't think of that.